app/templates/_component.js

```diff
@@ -108,3 +108,73 @@ require('jqueryui');
   });
   <% } %>
 })(window.jQuery, window, document);
+<% } else { %>'use strict';
+
+/**
+ *@fileOverview
+ *@version 0.0.1
+ *
+ * @namespace factorial.<%= _.camelCase(name) %>
+ */
+
+var _ = require('lodash');
+
+var defaults = {
+  value: true
+};
+
+var elements = {
```

Comment:
I don't fancy defining elements here. 

Should be part of the configuration options and passed as selectors, which is note self-evident from its variable name.